File: myExpenses/src/test/java/org/totschnig/myexpenses/provider/filter/OperationTest.java
package org.totschnig.myexpenses.provider.filter;

import junit.framework.Assert;
import junit.framework.TestCase;

public class OperationTest extends TestCase {

  public void testGetOp() {
    Assert.assertEquals("IN ()", WhereFilter.Operation.IN.getOp(0));
    Assert.assertEquals("IN (?)", WhereFilter.Operation.IN.getOp(1));
    Assert.assertEquals("IN (?,?)", WhereFilter.Operation.IN.getOp(2));
    Assert.assertEquals("IN (?,?,?)", WhereFilter.Operation.IN.getOp(3));

  }
}

File: myExpenses/src/test/java/org/totschnig/myexpenses/provider/filter/OperationTest.kt
package org.totschnig.myexpenses.provider.filter

import junit.framework.TestCase

class OperationTest : TestCase() {
    fun testGetOp() {
        assertEquals("IN ()", Operation.IN.getOp(0))
        assertEquals("IN (?)", Operation.IN.getOp(1))
        assertEquals("IN (?,?)", Operation.IN.getOp(2))
        assertEquals("IN (?,?,?)", Operation.IN.getOp(3))
    }
}